docs/deploying-applications/deploying-to-azure/deploying-a-package-to-an-azure-web-app/index.md

```diff
@@ -130,7 +130,9 @@ If you haven't already, create an [Azure Subscription Account](/docs/key-conce
 
 Once an Account is selected, the list of Azure Web Apps available to the subscription associated with the account will populate the 'Web App' select-list.
 
-![](/docs/images/3048686/3278366.png "width=500")
+If you choose to run this step on behalf of target roles (maybe you are deploying to multiple geographic regions), you will need to ensure a Deployment Target exists when deploying your Azure Web App. For this, we introduced [Cloud Regions](/docs/deployment-targets/cloud-regions). If you select a role and no Deployment Targets exist at the time of deploying, Octopus will log warnings in your deployment's task log.
```

Comment:
Links to other pages need to link to the md file, see https://github.com/octopusdeploy/docs#linking